File: frontend/src/app/models/result-answer/result-answer-from-result-dto.ts
import { ResultAnswerOptionDetailDto } from '../result-answer-option/result-answer-option-detail-dto';
import { ResultQuestionDto } from '../question/result-question-dto';

export interface ResultAnswerFromResultDto {
    id: number;
    timeTakenSeconds: string;

    question: ResultQuestionDto;
    resultAnswerOptions: ResultAnswerOptionDetailDto[];
}

File: QuizApp/src/QuizApp.BLL/Dto/ResultAnswer/ResultAnswerFromResultDto.cs
using System;
using System.Collections.Generic;
using System.Text;

using QuizApp.BLL.Dto.ResultAnswerOption;
using QuizApp.BLL.Dto.TestQuestion;

namespace QuizApp.BLL.Dto.ResultAnswer
{
    public class ResultAnswerFromResultDto
    {
        public int Id { get; set; }

        public TimeSpan TimeTakenSeconds { get; set; }


        public ResultQuestionDto Question { get; set; }

        public ICollection<ResultAnswerOptionDetailDto> ResultAnswerOptions { get; set; } = new List<ResultAnswerOptionDetailDto>();
    }
}
